node/state/core.go

```diff
@@ -31,9 +33,25 @@ func CoreAccessor(endpoint string) func(fx.Lifecycle, keystore.Keystore, params.
 			return nil, err
 		}
 		signer := apptypes.NewKeyringSigner(ring, keyringAccName, string(net))
+		keys, err := signer.List()
+		if err != nil {
+			return nil, err
+		}
+		// if no key was found in keystore path, generate new key for node
+		if len(keys) == 0 {
+			log.Infow("NO KEY FOUND IN PATH, GENERATING NEW KEY...", "path", ks.Path())
```

Comment:
```suggestion
			log.Infow("NO KEY FOUND IN STORE, GENERATING NEW KEY...", "path", ks.Path())
```
I know that we hardcode path right now for the key storage, but we shouldn't do so and tell users that we store key in the store of the node which can be anywhere users wants. 
(Now it makes sense to differentiate between store and repository btw)

Reply:
done